t/message-decode-brotlibomb.t

```diff
@@ -0,0 +1,104 @@
+# https://rt.cpan.org/Public/Bug/Display.html?id=52572
+
+use strict;
+use warnings;
+
+use Test::More;
+
+use HTTP::Headers    qw();
+use HTTP::Response   qw();
+
+my $ok = eval {
```

Comment:
I think we could use `Test::Needs` here. That would be a little less verbose.

Reply:
I've moved the manual checking/loading of modules into `Test::Needs` (which already is a prerequisite) for this and for `t/message-decode-zipbomb.t` as well.